rest/model/createhost.go

```diff
@@ -9,16 +9,44 @@ type CreateHost struct {
 	DNSName    string `json:"dns_name"`
 	IP         string `json:"ip_address"`
 	InstanceID string `json:"instance_id"`
+
+	ContainerID string `json:"container_id"`
+	Host        string `json:"host"`
+	Image       string `json:"image"`
+	Command     string `json:"command"`
+	Port        int    `json:"port"`
```

Comment:
If this is the port that the docker API is running on, this is a property of the parent rather than the container. Also we shouldn't expose this in the REST model